### ISSUE TITLE
 react/no-typos false error Typo in declared prop type for Immutable.js ImmutablePropTypes

### ISSUE THREAD

**Issue by JustFly1984** [yannickcr/eslint-plugin-react#1741]
I use immutable.js extensively accross my apps.
I get errors on ImmutablePropTypes.map and ImmutablePropTypes.list

```
Typo in declared prop type: map  react/no-typos
Typo in declared prop type: list  react/no-typos
```

Please help. Currently it breaks my build, so I will mute this rule until you fix it

**Comment by jseminck** [comment]
Can you show the code? A PR was merged recently that hopefully fixes this issue: https://github.com/yannickcr/eslint-plugin-react/pull/1671

**Comment by ljharb** [comment]
It's been merged.
